### DISCUSSION TITLE
I'm lost at sea with installation

### DISCUSSION THREAD

**Discussion by Mousesintheneighborhood** [apprenticeharper/DeDRM_tools#2244] [General]
I'm clueless about the installation that will let me read kindle books in calibre on my pc.  It has been years since I installed an old calibre and alf's drm tools  They worked fine until a few months ago when amazon disabled kindle for pc unless one was running at least windows 8.

So I brought up Windows 10, installed the latest calibre 6.20.0 and installed what seems to be the latest drm removal plug 7.2.1.

When I try to pull over amazon books I get a message somewhat like unknown book format.  The kindle directory name for the books ends in EBOK.

I gather from trying to find an answer that perhaps there are additional plugins I need?  Is there some simple set of directions somewhere for what I need to do besides what I have already done?  Thanks.

**Comment by ElleKayEm** [comment]
The Apprentices have retired.  Find the latest DeDRM 10.0.3 here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

To get books you can decrypt from Amazon, see this post for known workarounds: https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by Mousesintheneighborhood** [reply]
Forget this, I found the file :-)

Thanks very much.  Here's what I just did until I ran into a problem:

I didn't know how to remove the old Alf plug in, so I saved Calibre Library, uninstalled calibre, installed calibre, and copied the saved Calibre Library over the newly created one.  So far so good.

Then I downloaded DeDRM_tools_10.0.3.zip, which was all I could find on the new page.  I went through the plugin from file process and when I pointed  it at this file, it complained that a top level file was missing.  In the past the old stuff first failed this way also until I learned to point it at DeDRM_plugin.zip, then it worked.

So I am not sire what to do here, is DeDRM_plugin.zip lurking some place or?

Thank you.

**Reply by ElleKayEm** [reply]
Unzip DeDRM_tools_10.0.3.zip.  Inside you'll find the 2 plugins (Obok is for Kobo books) and 3 readme text files.  Load DeDRM_plug.zip into calibre.

**Comment by Mousesintheneighborhood** [comment]
I looked at the for Amazon hacks.  Based on the versions of the software I have, the disappearance of older kindle versions, no kindle ereader, I think I am out of luck, but I will look at it more later.

One thing that puzzles me is the reference to an error from the plug in complaining about a KFX error.  Perhaps that is some old stuff, as mine just said something like unknown book format.

The reason I would like to get the books on calibre is I discovered years ago that if I close my amazon account, amazon deletes all my kindle books on my pc.  I want to save them somewhere amazon can't get to them.

**Comment by ElleKayEm** [comment]
See the info in that MobileRead post about obtaining older Kindle for PC versions which will still work for books published before 2023.  There's also a method using an older Kindle for Android version which works for recently published books and doesn't require an Android device

**Comment by Mousesintheneighborhood** [comment]
I'm interested in new books, so that left me with -

https://www.mobileread.com/forums/showpost.php?p=4322408&postcount=308

which is kind of hairraising.  I don't have time to look at it in detail for awhile.

**Reply by laurenwatic** [reply]
Thanks for link

**Reply by Mousesintheneighborhood** [reply]
I started looking at it and got as far as below and then bailed.  The directions are very far beyond what I know and I thought I might mess up my laptop trying to do them:

https://www.mobileread.com/forums/showthread.php?t=283371

Method 6 - Use an old version of Kindle for Android

The Kindle for Android app can be run using emulator software on just about any system. An Android device is not required.

See the thread: DRM workaround: Kindle for Android for details.

That thread was perplexing, so it said to go to:
https://www.mobileread.com/forums/showpost.php?p=4322408&postcount=308
That had a downloadable file with directions.

It was the downloadable file that caused me to bail.

**Reply by Mousesintheneighborhood** [reply]
I took another try at this today and it is beyond incomprehensible.  So I am saving a copy of everything new I get via kindle for pc elsewhere, in hopes that in the future if I no longer have an amazon account I can disconnect from the web, copy whatever book I want into the kindle folder and read it.  No idea if this will work.  Also I really miss the well organized calibre library, kindle's just sucks.

**Reply by ElleKayEm** [reply]
No, that will not work.

The latest master code (not in a released version yet) at noDRM's repo should work on most books from Kindle for PC 1.40 (at least for the moment).  You can get it by clicking plugin at the bottom of this page: https://github.com/noDRM/DeDRM_tools/actions/runs/5370838206

Unzip that file and then the folder inside it.  Load DeDRM_plugin.zip into calibre as usual.  You will also need the KFX Input plugin:  https://www.mobileread.com/forums/showthread.php?t=291290

**Reply by Mousesintheneighborhood** [reply]
Thank you!

Should I remove the plugin that is already there (Apprentice Harper thing) first?  Also what does repo mean?

**Reply by ElleKayEm** [reply]
No, you don't need to remove the older DeDRM first.  Repo is short for repository here at GitHub.  The current DeDRM is from noDRM's repository which was originally created from Apprentice Harper's.

**Comment by Mousesintheneighborhood** [comment]
> No, that will not work.
> 
> The latest master code (not in a released version yet) at noDRM's repo should work on most books from Kindle for PC 1.40 (at least for the moment). You can get it by clicking plugin at the bottom of this page: https://github.com/noDRM/DeDRM_tools/actions/runs/5370838206
> 
> Unzip that file and then the folder inside it. Load DeDRM_plugin.zip into calibre as usual. You will also need the KFX Input plugin: https://www.mobileread.com/forums/showthread.php?t=291290

Things may have changed while I was off doing something else.  I think I installed the two plugins correctly just now (but maybe not) and then I dragged over the azw file of a book I had purchased and can read via kindle for pc from My Kindle Content to calibre.  Unlike before this where if I remember the error message correctly calibre was mumbling about an unknown format, this time it said to look for an app in the microsoft store and always use that app to open kfx files.  Might that actually work if I can find such an app, or is this trying to say the kfx plugin is not happy.

**Reply by ElleKayEm** [reply]
If you're getting kfx and not kfx-zip, sounds like the drm is removed.  Set the calibre ebook viewer to open kfx format.  Or try converting to another format.

**Reply by Mousesintheneighborhood** [reply]
Yes!!  I converted it to epub and now I can read it!  Thank you!
